### PR TITLE
Enable utility feature

### PR DESCRIPTION
**Pull request by sabf-tutorials**
This is the initial implementation of the utility feature based on the requirements.

### CODE REVIEW

File: util/config.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+config1:abc
+
+config2:123
```
Comment (sabf-main) [inline util/config.txt]: This config value should be 567
The old value was 123

File: util/feature.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+feature 1 implementation
+
+feature 2 implementation
```
Comment (sabf-main) [inline util/feature.txt:3]: Let's add a bit more to the feature2 implementation. I think it should handle multiple widgits

**Review by sabf-main — CHANGES_REQUESTED** [review]
Please fix the config values and add the implementation to feature2

File: util/feature.txt (hunk 2 above)
Comment (sabf-tutorials) [reply util/feature.txt:3]: @sabf-main see the latest commit for the implementation

**Review by sabf-tutorials — COMMENTED** [review]
Modified code based on review

**Review by sabf-main — APPROVED** [review]

**Pull request merged**